SPECS/docker-compose/docker-compose.spec

```diff
@@ -0,0 +1,56 @@
+Summary:        Define and run multi-container applications with Docker
+Name:           docker-compose
```

Comment:
We generally name things `moby-*` even if they come from different sources like docker. (see `containerd` and `runc` etc.)